.github/workflows/dotnet-core.yml

```diff
@@ -19,10 +19,21 @@ jobs:
       with:
         fetch-depth: 0 # fetching all
 
-    - name: Setup .NET Core
+    - name: Setup .NET Core 2.1.807
```

Comment:
I think name can be just "Setup .NET Core 2.1" without mentioning the exact patch version.

Reply:
ok!